### PR TITLE
Adjust styles to fit github changes

### PR DESCRIPTION
**Pull request by educalleja**
Related issue [#127 ](https://github.com/berzniz/github_pr_tree/issues/127).

Updates markup to render the extension correctly.

Sorry, I was not able to test in Firefox.

### CODE REVIEW

**Comment by lingzhang000** [comment]
LGTM, thanks for making this change

File: src/js/lib.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 export const createRootElement = () => {
-  const injectionElement = document.querySelector('.pr-toolbar')
+  const injectionElement = document.querySelector('#files_bucket')
+  const nextSibiling = document.querySelector('.diff-view')
```
Comment (homburg) [inline src/js/lib.js:3]: Typo, awesome work anyway 👏 

```suggestion
  const nextSibling = document.querySelector('.diff-view')
```

File: src/js/lib.js [hunk 2]
```diff
@@ -8,7 +9,7 @@ export const createRootElement = () => {
   if (!element) {
     element = document.createElement('div')
     element.className = rootId
-    injectionElement.appendChild(element)
+    injectionElement.insertBefore(element, nextSibiling)
```
Comment (homburg) [inline src/js/lib.js:12]:
```suggestion
    injectionElement.insertBefore(element, nextSibling)
```

**Review by homburg — CHANGES_REQUESTED** [review]

**Comment by berzniz** [comment]
Hi - thank you for the PR.

Testing and uploading to the store.

**Comment by berzniz** [comment]
Thanks for your PR. I decided not to merge it due to it changing the way the tree-view behaves. With your suggested fix, it looks great, but scrolls along with the page.

I opted for a different solution (#129) but will review this solution again when there is more time. Currently the extension is rendered useless and I'd like to provide a quick-fix for it.

**Pull request closed without merging**